HW1/P6/P6.py

```diff
@@ -0,0 +1,109 @@
+import random
+
+import findspark
+findspark.init('/home/lhoang/spark')
+
+import pyspark
+sc = pyspark.SparkContext(appName="spark1")
+
+
+def get_gram(v):
```

Comment:
I guess you could also have mapped each triplet to 1, and the reduce to get the number of appearances of each triplet.